Makefile

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+install-dev:
+	pip3 install -e .[dev]
+	curl -fLSs https://raw.githubusercontent.com/CircleCI-Public/circleci-cli/master/install.sh | bash
+
+bump-minor:
+	bump2version minor
```

Comment:
It gets installed via the `make install-dev` command as well as the CircleCI command line tools which are necessary since they're called to verify the CircleCI config during the commit stage as a pre-commit hook.